src/content/docs/en/guides/upgrade-to/v3.mdx

```diff
@@ -524,7 +524,15 @@ Astro v3.0 now includes Sharp as the default image processing service and instea
 
 #### What should I do?
 
-If you would prefer to continue to use Squoosh to transform your images, update your config with the following: 
+:::note
+When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like `pnpm`, you may also need to manually install Sharp into your project:
```

Comment:
```suggestion
When using a [strict package manager](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) like `pnpm`, you may need to manually install Sharp into your project even though it is an Astro dependency:
```
I took out "also" because there's no other step mentioned first, replacing it with an "EVEN THOUGH" to address any doubt!